opensrp-reveal/src/main/java/org/smartregister/reveal/sync/RevealClientProcessor.java

```diff
@@ -201,4 +222,13 @@ public String calculateBusinessStatus(Event event) {
         }
     }
 
+    private String calculateMosquitoCollectionTaskBusinessStatus(Event event) {
```

Comment:
Business status should be calculated on form, so that its possible to have different calculations for different countries depending on form.

The field key for the business status  on the form should be uniform, so that we maintain one business status calculation.

e.g the below should be able to work for all forms going further
 Obs taskBusinessStatusObs = event.findObs(null, false, JsonForm.BUSINESS_STATUS);

Reply:
Done.